src/server/utilities/cspHeader/index.js

```diff
@@ -343,6 +343,7 @@ const helmetCsp = ({ isAmp, isLive }) => ({
     'script-src': generateScriptSrc({ isAmp, isLive }),
     'style-src': generateStyleSrc({ isAmp, isLive }),
     'worker-src': generateWorkerSrc({ isAmp }),
+    'report-to': 'default',
```

Comment:
might be worth confirming if we want this value for all environments (non-live/live) and platforms (amp/canonical)

Reply:
Reporting on test is fine. so looks like this PR is good to go.